DotnetViewComponents/Views/Shared/Components/CharacterCount/Default.cshtml

```diff
@@ -0,0 +1,30 @@
+@using DotnetViewComponents.ViewModels
+@model CharacterCountViewModel
+
+
+<div class="nhsuk-character-count" data-module="nhsuk-character-count"
+@(Model.MaxWords != null ? $"data-maxwords={Model.MaxWords}" : $"data-maxlength={Model.MaxLength}") @(Model.Threshold != null ? $"data-threshold={Model.Threshold}" : "")>
+
+    <div class="nhsuk-form-group @(Model.HasError ? "nhsuk-form-group--error" : "")">
+        <h1 class="nhsuk-label-wrapper">
+            <label class="nhsuk-label nhsuk-label--l" for=@(Model.Id)>
+                Can you provide more detail?
+            </label>
+        </h1>
+        <div class="nhsuk-hint" id="@(Model.Id)-hint">
+            Do not include personal information like your name, date of birth or NHS number.
+        </div>
+        @if (Model.HasError)
+        {
+            <span class="nhsuk-error-message" id="@(Model.Id)-error">
+                <span class="nhsuk-u-visually-hidden">Error:</span> Job description must be 350 characters or less
```

Comment:
we should used the MaxLength param in place of hard-coded "350" here, I think. We ought to be handling some of the MaxWords, MaxLength and Threshold params here too?